test/features/10-functional/20-personalisation/20-location-search.feature

```diff
@@ -24,6 +24,6 @@ Feature: Location search
         And the button "Done" should be enabled
 
     Scenario: Suburb search normalises spaces
-        When I visit /personalise/location
+        When I visit /category/housing/personalise?subpage=1
```

Comment:
In fact, I reordered the personalisation questions in my current branch, which broke things so it's worth fixing.